### ISSUE TITLE
Question about augmentation during eval-train

### ISSUE THREAD

**Issue by MrChenFeng** [KentoNishi/Augmentation-for-LNL#4]
Hi,

Thanks so much for your great work! After reading your paper, I would expect you used a weak augmented image during eval-train step, however I noticed you used no transformation for this in the code. And the only difference to dividemix for me is that you used a somewhat augmentation anchoring trick[ReMixmatch] in training step. Thanks so much if you could confirm that if I have some misunderstandings?

Best,
Chen

**Comment by KentoNishi** [comment]
Thanks for reaching out!

I re-read my code to be sure, and your analysis seems correct. This is a genuine mistake on my part. Although the lack of augmentation in `eval_train` should have negligible impact in practice, `eval_train` is supposed to be weakly augmented. In this implementation of `AugDesc-WS` for DivideMix, the operation is very similar to augmentation anchoring.

I re-ran the code (simply replacing the augmentation for `eval_train` with weak augmentations) for `c10.90sym.AugDesc-WS`, and the accuracy was 90.280. I have attached the training logs below. The overall contribution of the paper is the comparison of multiple strategies and the observation that separating clean and noisy augmentation pools for different tasks is favorable, and we believe this analysis remains valid.

I will link this issue in the code for future readers. (edit: I added a comment in commit https://github.com/KentoNishi/Augmentation-for-LNL/commit/298495499ad51084931e31cd28525427f7444412)

Please let me know if you have any further questions!

[metrics.log](https://github.com/KentoNishi/Augmentation-for-LNL/files/7095147/metrics.log)
